permissions/plugin-pipeline-config.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "pipeline-model-definition"
+paths:
+- "org/jenkinsci/plugins/pipeline-model-definition"
```

Comment:
Yup. My instincts are always to avoid the dash in groupId. I can change it if you'd prefer.